### PR TITLE
fix(Poll) - hide voters number from open poll if user has not voted

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Currently, when user hasn't voted yet in the poll, server returns voters number as 0 always. See reference: https://github.com/nextcloud/spreed/blob/9b525c2229d73e6989a7182949fa45fdbc84f457/lib/Controller/PollController.php#L245-L256
* PR aims to align frontend behavior with backend - hide poll voters number for people who are not moderators/owners/voted

### 🖼️ Screenshots

🏚️ Before (admin) | 🏚️ Before (user) | 🏡 After (admin) | 🏡 After (user)
---|---|---|---
image | image | image | image
image | image | image | image
image | image | image | image
image | image | image | image
image | image | image | image
image | image | image | image
image | image | image | image

### 🚧 Tasks

- [ ] Visual check
- [ ] Code review
- [ ] Change `Poll` to `Open poll` ???

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue [hunk 1]
```diff
@@ -315,61 +308,46 @@ export default {
...
+			return this.$store.getters.conversation(this.token).participantType
 		},
 
-		/**
-		 * Current actor type
-		 */
-		actorType() {
-			return this.$store.getters.getActorType()
+		selfIsOwnerOrModerator() {
+			return (this.poll.actorType === this.$store.getters.getActorType() && this.poll.actorId === this.$store.getters.getActorId())
```
Comment (ShGKme) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue:316]: Now I feel, we need an util function for this :D

**Review by ShGKme — COMMENTED** [review]
Looks good by screenshots and the code, but haven't tested manually yet

**Review by ShGKme — APPROVED** [review]
Tested, works good, same as on screenshots

**Comment by ShGKme** [comment]
> Change Poll to Open poll ???

Where?

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/Poll.vue:316]: Maybe a getter? I tried to implement it, as `selfIsActor`, and it shows itself pretty good. Though we have a different checks in different places for this, so should be unified first

**Comment by Antreesy** [comment]
> Where?

For polls which are not closed yet, like this:
`Open poll ° 3 votes`

**Review by marcoambrosini — APPROVED** [review]
Not tested but screenshots look good. Agree with shortening the text.

**Comment by ShGKme** [comment]
> For polls which are not closed yet, like this:
> `Open poll ° 3 votes`

Yes, that makes sense to me

**Pull request merged**